### PR TITLE
Support schema.examples in Form

### PR DESCRIPTION
**Pull request by sradevski**
Change-type: minor
Signed-off-by: Stevche Radevski <stevche@balena.io>

Screen Shot 2020-03-23 at 13 54 53

Screen Shot 2020-03-23 at 13 55 00
---
##### Contributor checklist

- [ ] I have rebuilt the README with `npm run build:docs`
- [ ] I have regenerated screenshots for any affected components with `npm run generate-screenshots`
- [ ] I have regenerated jest snapshots for any affected components with `npm run jest -- -u`

##### Reviewer Guidelines
- When submitting a review, please pick:
  - '*Approve*' if this change would be acceptable in the codebase (even if there are minor or cosmetic tweaks that could be improved).
  - '*Request Changes*' if this change would not be acceptable in our codebase (e.g. bugs, changes that will make development harder in future, security/performance issues, etc).
  - '*Comment*' if you don't feel you have enough information to decide either way (e.g. if you have major questions, or you don't understand the context of the change sufficiently to fully review yourself, but want to make a comment)
---

### CODE REVIEW

File: src/unstable/components/Form/widgets/BaseInput.tsx [hunk 1]
```diff
@@ -28,13 +36,18 @@ const BaseInput = (props: FormWidgetProps) => {
...
+
 	return (
 		<Input
 			width="100%"
 			readOnly={readonly}
 			disabled={disabled}
 			autoFocus={autofocus}
-			autoComplete={options.autocomplete}
+			// Form suggestions will clash with browser suggestions if autocomplete is on
+			autoComplete={
```
Comment (iamsolankiamit) [inline src/unstable/components/Form/widgets/BaseInput.tsx]: autoComplete off doesn't work on multiple versions of chrome. I have found that adding a blank `<datalist />` does disables the browsers autocomplete.

Comment (sradevski) [reply src/unstable/components/Form/widgets/BaseInput.tsx]: 👍

File: src/unstable/components/Form/story.js [hunk 2]
```diff
@@ -62,7 +62,8 @@ const basicPokedexSchema = {
   properties: {
     Name: {
       type: 'string',
-      minLength: 5
+      minLength: 5,
+      examples: ['Pikachu', 'Snorlax', 'Charmander', 'Bulbasaur']
```
Comment (LucianBuzzo) [inline src/unstable/components/Form/story.js]: Is this in the JSON schema spec?

Comment (sradevski) [reply src/unstable/components/Form/story.js]: Yes, https://rjsf-team.github.io/react-jsonschema-form/

Comment (iamsolankiamit) [reply src/unstable/components/Form/story.js]: @LucianBuzzo yes

**Pull request merged**